examples/basic/layouts/dashboard.py

```diff
@@ -33,59 +30,55 @@ def slider():
 
     source = ColumnDataSource(data=dict(x=x, y=y))
 
-    plot = figure(
-        y_range=(-10, 10), tools='', toolbar_location=None,
-        title="Sliders example")
+    plot = figure(y_range=(-10, 10), tools='', toolbar_location=None,
+                  title="Sliders example")
     plot.line('x', 'y', source=source, line_width=3, line_alpha=0.6)
 
-    amp_slider = Slider(start=0.1, end=10, value=1, step=.1, title="Amplitude")
-    freq_slider = Slider(start=0.1, end=10, value=1, step=.1, title="Frequency")
-    phase_slider = Slider(start=0, end=6.4, value=0, step=.1, title="Phase")
-    offset_slider = Slider(start=-5, end=5, value=0, step=.1, title="Offset")
+    amp = Slider(start=0.1, end=10, value=1, step=.1, title="Amplitude")
+    freq = Slider(start=0.1, end=10, value=1, step=.1, title="Frequency")
+    phase = Slider(start=0, end=6.4, value=0, step=.1, title="Phase")
+    offset = Slider(start=-5, end=5, value=0, step=.1, title="Offset")
 
-    callback = CustomJS(args=dict(source=source, amp=amp_slider, freq=freq_slider, phase=phase_slider, offset=offset_slider),
+    callback = CustomJS(args=dict(source=source, amp=amp, freq=freq, phase=phase, offset=offset),
                         code="""
-        const data = source.data;
         const A = amp.value;
         const k = freq.value;
         const phi = phase.value;
         const B = offset.value;
-        const x = data['x']
-        const y = data['y']
-        for (let i = 0; i < x.length; i++) {
-            y[i] = B + A*Math.sin(k*x[i]+phi);
-        }
-        source.change.emit();
+
+        const x = source.data.x
+        const y = Array.from(x, (x) => B + A*Math.sin(k*x+phi))
+        source.data = { x: x, y: y }
```

Comment:
This could use JS shorthand syntax and no spaces for consistency with bokehjs:
```suggestion
        source.data = {x, y}
```

Reply:
https://github.com/bokeh/bokeh/pull/12490/commits/2b8f930cad244905161ff5857d3d3fe8048039f4